python/py-cryptography/Portfile

```diff
@@ -1,11 +1,13 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           cargo_fetch 1.0
+PortGroup           github 1.0
 PortGroup           python 1.0
 PortGroup           compiler_blacklist_versions 1.0
```

Comment:
are you actually using this PG?

Reply:
“`cargo.crates`” avoids rust fetches after the fetch phase.

Reply:
sorry for not being completely clear, I was referring to the `compiler_blacklist_versions` PG

Reply:
I still don't see where the `compiler_blacklist_versions` PG is used in that subport either; I might be missing it, so please point me to it

Reply:
Removed thanks.